snap/info_snap_yaml.go

```diff
@@ -223,11 +223,13 @@ func setPlugsFromSnapYaml(y snapYaml, snap *Info) error {
 			return err
 		}
 		snap.Plugs[name] = &PlugInfo{
-			Snap:      snap,
-			Name:      name,
-			Interface: iface,
-			Attrs:     attrs,
-			Label:     label,
+			PlugSlotData{
```

Comment:
this kind of change looks a bit odd tbh, given that is all exported and not some internal detail